modules/gitops-argocd-cli-logging-in-to-argocd-server.adoc

```diff
@@ -0,0 +1,62 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/logging-in-to-argocd-server.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argocd-cli-logging-in-to-argocd-server{context}"]
+= Logging in to the Argo CD server
+
+After you install and configure the {gitops-shortname} `argocd` CLI, you must log in to the Argo CD server to execute commands in the default mode.
+
+.Prerequisites
+
+* You must have the {gitops-shortname} `argocd` CLI tool installed and configured.
+
+.Procedure
+
+You must use your credentials to log in and remain logged in throughout the session. 
```

Comment:
Should this be an important callout? You could even include it in the tip on line 21 maybe?

Reply:
thanks for the comment. I am retaining it as is.